redfish-core/lib/environment_metrics.hpp

```diff
@@ -117,58 +115,100 @@ inline void
 inline void getPowerWatts(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
```

Comment:
I think we need to relook at the dbus calls here. I think we might be able to eliminate the outer dbus call and replace it with an error not found for total_power.